README.md

```diff
@@ -62,9 +74,9 @@ To start automatic rebalancing with the Cellars rebalancer, run the start comman
 CELLAR_DRY_RUN=false cargo run -- -c [your_config_file_name.toml] start
```

Comment:
In this section, we should remove references to the poller which no longer exists.

Reply:
fixed